packages/venia-ui/lib/components/CheckoutPage/PaymentInformation/summary.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { shape, string, func } from 'prop-types';
+import { Edit2 as EditIcon } from 'react-feather';
+import { useSummary } from '@magento/peregrine/lib/talons/CheckoutPage/PaymentInformation/useSummary';
+
+import Icon from '../../Icon';
+import { mergeClasses } from '../../../classify';
+
+import summaryGQLOperations from './summary.gql';
+
+import defaultClasses from './summary.css';
+
+const Summary = props => {
+    const { classes: propClasses, onEdit } = props;
+
+    const classes = mergeClasses(defaultClasses, propClasses);
+
+    const talonProps = useSummary(summaryGQLOperations);
```

Comment:
I am sure the future developers working on this will know that the prop takes an object and they can send more if needed. Not to sound dramatic but this is kind of stuff they need to know to work on our/any codebase. But if that is a standard we want to stick to, ill change.

Reply:
This is such a minor change, and we do it in the rest of the app. Please make the change.